src/emulator/types.ts

```diff
@@ -1,21 +1,40 @@
 import { ChildProcess } from "child_process";
 import { EventEmitter } from "events";
+import * as path from "path";
 
 export enum Emulators {
```

Comment:
I think enums are usually better off defined in the singular, but...

Reply:
Yeah I regret this from the start but don't want to bloat the diff changing it now.